src/class_addressGenerator.py

```diff
@@ -248,12 +246,12 @@ def run(self):
                     while True:
                         numberOfAddressesWeHadToMakeBeforeWeFoundOneWithTheCorrectRipePrefix += 1
                         potentialPrivSigningKey = hashlib.sha512(
-                            deterministicPassphrase +
-                            encodeVarint(signingKeyNonce)
+                            deterministicPassphrase
+                            + encodeVarint(signingKeyNonce)
                         ).digest()[:32]
                         potentialPrivEncryptionKey = hashlib.sha512(
-                            deterministicPassphrase +
```

Comment:
make this into a function, `priv_key_from_passphrase`

Reply:
Will update in next PR